data/CA/incentives.json

```diff
@@ -0,0 +1,2817 @@
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "ca_bayAreaRegionalEnergyNetwork(BayREN)_rebates:SingleFamilyHomeowners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 150
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Get $150 cash back on air sealing when you work with a BayREN Participating Contractor."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "CA-123",
+    "authority_type": "other",
+    "authority": "ca-bay-area-regional-energy-network-bay-ren",
+    "eligible_geo_group": "ca-group-bay-area-regional-energy-network-bay-ren",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "ca_bayAreaRegionalEnergyNetwork(BayREN)_rebates:SingleFamilyHomeowners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$500 rebate for duct replacement."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "CA-124",
+    "authority_type": "other",
+    "authority": "ca-bay-area-regional-energy-network-bay-ren",
+    "eligible_geo_group": "ca-group-bay-area-regional-energy-network-bay-ren",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "ca_bayAreaRegionalEnergyNetwork(BayREN)_rebates:SingleFamilyHomeowners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 400,
+      "minimum": 250,
+      "maximum": 400
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Get $400 cash back on a heat pump installation when you work with a BayREN Participating Contractor."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "CA-125",
+    "authority_type": "other",
+    "authority": "ca-bay-area-regional-energy-network-bay-ren",
+    "eligible_geo_group": "ca-group-bay-area-regional-energy-network-bay-ren",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_clothes_dryer",
+    "program": "ca_bayAreaRegionalEnergyNetwork(BayREN)_rebates:SingleFamilyHomeowners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Get $250 back on your high efficiency heat pump clothes dryer when replacing an existing natural gas clothes dryer."
+    }
+  },
+  {
+    "id": "CA-126",
+    "authority_type": "utility",
+    "authority": "ca-riverside-public-utilities",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "ca_riversidePublicUtilities_residentialRebates",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 0.2,
+      "unit": "square_foot"
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Attic Insulation upto 20¢ per sq. ft. Minimum Rating R-19."
```

Comment:
"up to", and let's go with $0.20 to be consistent. (I'd actually consider switching all dollar amounts less than 1 to be expressed as ¢, but I'd rather do that as an across-the-board change and not have this as a one-off.)